plugin/helper/writer.go

```diff
@@ -6,32 +6,32 @@ import (
 	"fmt"
 
 	"github.com/observiq/carbon/entry"
-	"github.com/observiq/carbon/errors"
 	"github.com/observiq/carbon/plugin"
 )
 
 // WriterConfig is the configuration of a writer plugin.
 type WriterConfig struct {
-	OutputIDs OutputIDs `json:"output" yaml:"output"`
+	BasicConfig `yaml:",inline"`
```

Comment:
I like this change a lot. It makes sense that all other plugin types should embed Basic.